LearningMissionSimulation/PlaygroundEdvard.cs

```diff
@@ -107,5 +119,69 @@ public void ActivateAccounts()
 
             Console.WriteLine("-----Unfair coordinator finished activating accounts-----");
         }
+
+        public void CreateSubjects(int subjectCount)
+        {
+            for (int i = 0; i < subjectCount; i++)
+            {
+                Subject subject = ObjectGenerator.GenerateSubject();
+
+                subjectDictionary.Add(subject.Id, subject);
+                subjectIdList.Add(subject.Id);
+            }
+        }
+
+        public void CreateModules(int moduleCount)
+        {
+            for (int i = 0; i < moduleCount; i++)
+            {
+                Guid subjectId = GetSubjectId();
+
+                Module module = ObjectGenerator.GenerateModule(subjectId);
+
+                moduleDictionary.Add(module.Id, module);
+                subjectIdList.Add(module.Id);
+            }
+        }
+
+        Guid GetSubjectId()
+        {
+            return subjectIdList[AttributeGenerator.random.Next(0, subjectIdList.Count)];
+        }
+
+        public void AssignModulesToStudents()
+        {
+            foreach (Student student in studentList)
+            {
+                student.CompletedModuleList = GenerateModuleList();
+            }
+        }
+        
+        public void AssignModulesToInstructors()
+        {
+            foreach (Instructor instructor in instructorList)
+            {
+                instructor.ModuleList = GenerateModuleList();
+            }
+        }
+
+        public List<Module> GenerateModuleList()
+        {
+            List<Module> modules = new List<Module>();
+            int moduleCount = AttributeGenerator.random.Next(1, moduleDictionary.Count);
```

Comment:
1. Please use moduleListId.Count instead of moduleDictionary.Count. Even though they should give you the same value, it is an error prone approach. You are using this count to retrieve values from moduleIdList, therefore better use that data structure's  Count property.
2. Please view second part of the today's recording, when we discuss with Mher and Gavril the approaches to the module count value. Your current implementation allows adding to the instructor's or student's module lists as many modules as it is available in the moduleDictionary. But I want you to provide separate limits for the person's module list size. For example, you want to select from the available modules (say we have 100 generated modules)  but the list should contain no less than 2 and  no more than 5 modules.